addons/test_mass_mailing/tests/test_mailing_statistics_sms.py

```diff
@@ -43,13 +68,28 @@ def test_mailing_statistics_sms(self):
         for record_idx in (0, 2, 3):
             self.gateway_sms_click(mailing, target_records[record_idx])
 
-        # check mailing statistics
+        for record_idx in [7, 8]:
+            record = target_records[record_idx]
+            trace = mailing.mailing_trace_ids.filtered(lambda t: t.model == record._name and t.res_id == record.id)
+            trace.set_bounced()
+
+        for record_idx in [9, 10, 11, 12]:
+            record = target_records[record_idx]
+            trace = mailing.mailing_trace_ids.filtered(lambda t: t.model == record._name and t.res_id == record.id)
+            trace.set_canceled()
```

Comment:
Hello, would be better to have real canceled traces, due to invalid / duplicate number for example. always better to test a functional flow if possible